### DISCUSSION TITLE
Adapter design discussion

### DISCUSSION THREAD

**Discussion by KrutavShah** [KrutavShah/PCIE-to-MXM-Adapter#3] [General]
Moving from issue #1 to here as it is a discussion.

**Comment by chraac** [comment]
I found some others dc-dc integrated convertor for 12v to 5v with sop-8 package:
[RT8283A](https://www.richtek.com/assets/product_file/RT8283A/DS8283A-01.pdf)
[TS2509](https://www.mouser.com/datasheet/2/395/TS2509_B09-1918624.pdf)
[AP1538](https://www.mouser.com/datasheet/2/115/AP1538-69565.pdf)

[TPS54560](https://www.ti.com/lit/ds/symlink/tps54560.pdf?ts=1620415866393) from TI, 5A max

**Reply by KrutavShah** [reply]
I should also mention that we should keep costs low, and some components may cost a lot more on some sites than others. I am looking for some solutions right now that are a bit cheaper and can still carry out the same functionality.

**Reply by chraac** [reply]
The [TPS54328DDA](https://www.digikey.com/en/products/detail/texas-instruments/TPS54328DDAR/2503767) is selling on DigiKey with 1 for $1.4 and 25 for $29.68, maybe it's cheap enough for our adapter.

**Reply by KrutavShah** [reply]
Definitely better but there are only very few in stock and Texas Instruments may have stopped production of it in order to focus on a newer one. I am also looking into some alternative solutions that will be readily available at a reasonable price. $1 and less is the best price to stay at or under in order to match the inexpensive build of this adapter.

Another challenge we are likely to face is the limited availability of MXM connectors as the stock for those is drying up pretty fast.

**Reply by chraac** [reply]
Yep, Looks the [BD9D321EFJ](https://www.digikey.com/en/products/detail/rohm-semiconductor/BD9D321EFJ-E2/4625913) you mention before is our best choice for it. And for the MXM connector, I can just find [1 vendor](https://www.digikey.com/en/products/detail/yamaichi-electronics/CNU113-314-2201-VE/11681592) in DigiKey which is available to sell, maybe we should try another site to get the connector.

**Reply by KrutavShah** [reply]
> BD9D321EFJ

This one will actually end up being more expensive in the big picture as it requires a lot of external components such as resistors to function. There are other solutions out there for just a little bit more that only need 4 capacitors and an inductor. As for MXM connectors... well they cost more than the whole PCB. JAE still sells their remaining stock of connectors on Mouser for ~$7, and I don't know if they plan on producing more.

**Reply by chraac** [reply]
I did known foxconn produce the MXM connector, but it's for massive order, may not for retail.

**Reply by KrutavShah** [reply]
That's correct, not for retail. MXM is a dying standard anyway, soldering everything to laptop PCB is the future. I'm sure you could probably find MXM connectors on Aliexpress if you looked hard enough since they seem to have a lot of parts. Other than that, there are a few in stock and that should be fine for now. I'm going to make a final decision on which DC-DC buck converter to use and then I can figure out how to route HDMI. After that, it should be ready for final production.

**Reply by KrutavShah** [reply]
https://www.digikey.com/en/products/detail/diodes-incorporated/AP62300TWU-7/12702558

So I found this one, and it uses 6 pins and has a very small package. It looks like this will be the one to use for now, unless I can find a better alternative.

**Reply by chraac** [reply]
![image](https://user-images.githubusercontent.com/1937740/120764411-b2073280-c54a-11eb-90b3-5febbdc4b742.png)

The typical application circuit look simple, but there is 0 in stock in digikey and 3000pcs for minimum order to the manufacturer.

**Reply by KrutavShah** [reply]
Unfortunately I added it to the design a few weeks ago, when there was stock. Now they have 5 pieces remaining for order with a manufacturer shipment of 2022. 

With that said, do you know of any other components that have a low BOM/simple circuit? I’ll be looking for some as well. Thanks for letting me know.

**Reply by chraac** [reply]
Looks the [TPS563202](https://www.digikey.com/en/products/detail/texas-instruments/TPS563202SDRLR/13542531) is a better replacement for AP62300.
![image](https://user-images.githubusercontent.com/1937740/120895838-612e3180-c651-11eb-90f2-2befea1fd5a5.png)

**Reply by KrutavShah** [reply]
Thanks! I’ll try this new part in the design and see if it’s a good fit.

**Comment by 3aTup0k** [comment]
You can also remove the current sensor and replace it with a fuse.

**Reply by 3aTup0k** [reply]
and also add a switch from a built-in thermal sensor to an external one.  It will be a pleasure to help with development.

**Comment by Behamut** [comment]
I love the project and thank you so much for doing this. I've recently been looking for the reverse of this adapter, one that converts MXM3.0 to PCIe. I've found pictures and references of them but can't seem to locate that cable save for some discussion about the idea from an Imac thread of the latge 2011's. They used an mxm3.0a and my equipment is the 3.0b so not sure it'll help but I'm trying to upgrade my laptop's graphics card which will require vbios trickery and it would be nice to perhaps try it first with a changeable pcie normal card but thanks for the hard work

**Comment by KrutavShah** [comment]
Hi,Please refer to the following project:https://github.com/a-little-wifi/mxm-immobilizerThank you,Krutav ShahOn Sep 19, 2025, at 8:43 PM, Behamut ***@***.***> wrote:﻿
I love the project and thank you so much for doing this. I've recently been looking for the reverse of this adapter, one that converts MXM3.0 to PCIe. I've found pictures and references of them but can't seem to locate that cable save for some discussion about the idea from an Imac thread of the latge 2011's. They used an mxm3.0a and my equipment is the 3.0b so not sure it'll help but I'm trying to upgrade my laptop's graphics card which will require vbios trickery and it would be nice to perhaps try it first with a changeable pcie normal card but thanks for the hard work
